apps/site/assets/ts/stop/components/DepartureCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from "react";
+import { routeBgClass, busClass } from "../../helpers/css";
+import renderFa from "../../helpers/render-fa";
+import { breakTextAtSlash } from "../../helpers/text";
+import { isASilverLineRoute } from "../../models/route";
+import { Route } from "../../__v3api";
+import CRsvg from "../../../static/images/icon-commuter-rail-default.svg";
+import Bussvg from "../../../static/images/icon-bus-default.svg";
+import SubwaySvg from "../../../static/images/icon-subway-default.svg";
+import FerrySvg from "../../../static/images/icon-ferry-default.svg";
+import renderSvg from "../../helpers/render-svg";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const TypeToModeIcon = (route: Route): any => {
+  switch (route.type) {
+    case 0:
+    case 1:
+      return SubwaySvg;
+
+    case 2:
+      return CRsvg;
+
+    case 3:
+      return Bussvg;
+
+    case 4:
+      return FerrySvg;
+
+    default:
+      return null;
+      break;
+  }
+};
+
+const DepartureCard = ({
+  route
+}: {
+  route: Route;
+}): ReactElement<HTMLElement> => {
+  const RouteName = (
```

Comment:
Since this isn't a React component, I think camelcase naming would be better here.